components/web/icedtea-web/files/README

```diff
@@ -0,0 +1 @@
+https://github.com/mozilla/npapi-sdk
```

Comment:
Do we want this? If this required for the Firefox plugin, then I'd drop this as the plugin won't work anyway.